core/src/views/Profile.vue

```diff
@@ -68,7 +68,8 @@
 								class="user-actions__primary"
 								:href="primaryAction.target"
 								:icon="primaryAction.icon"
-								:target="primaryAction.id === 'phone' ? '_self' :'_blank'">
+								:target="primaryAction.id === 'phone' ? '_self' :'_blank'"
+								:rel="primaryAction.id === 'fediverse' ? 'me': null">
```

Comment:
```suggestion
								:rel="primaryAction.id === 'fediverse' ? 'me': undefined">
```

I think this makes it more clear and we use it like this on other places